core/streamer.py

```
@@ -156,8 +238,38 @@ def on_create_session_done(self, *args, **kwargs):
+            while self._watchdog_active:
+                healthy = check_health(self.proxy_url)
+                tripped = tracker.record(healthy, time.monotonic())
+                if tripped and not self._fail_closed_triggered:
+                    self._fail_closed_triggered = True
+                    print(
+                        "[FAIL_CLOSED] proxy /health fail-closed 지속"
+                        f" — 측정 중단함 (subject {self.subject_index})"
+                    )
+                    try:
+                        self.close_session()
+                    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)